src/components/split-button/split-button.interactions.stories.tsx

```diff
@@ -248,36 +289,175 @@ export const SizeVariations: Story = {
       await userInteractionPause(300);
     }
   },
-  decorators: [(StoryToRender) => <DefaultDecorator><StoryToRender /></DefaultDecorator>],
+  decorators: [
+    (StoryToRender) => (
+      <DefaultDecorator>
+        <StoryToRender />
+      </DefaultDecorator>
+    ),
+  ],
 };
 SizeVariations.storyName = "Sizes";
 
 export const Accessibility: Story = {
   render: () => (
     <Wrapper>
       <SplitButton buttonType="primary" text="Split button">
-        <Button data-role="target" aria-label="Save">Save</Button>
+        <Button data-role="target" aria-label="Save">
+          Save
+        </Button>
         <Button iconType="info" aria-label="More Info" />
       </SplitButton>
     </Wrapper>
   ),
   play: async ({ canvasElement }) => {
     if (!allowInteractions()) return;
     const canvas = within(canvasElement);
-    const toggleButton = canvas.getByRole('button', { name: /show more/i });
+    const toggleButton = canvas.getByRole("button", { name: /show more/i });
+
+    expect(toggleButton).toHaveAttribute("aria-expanded", "false");
+    expect(toggleButton).toHaveAttribute("aria-controls");
+    expect(toggleButton).toHaveAttribute("aria-label");
+
     await userEvent.click(toggleButton);
     await userInteractionPause(600);
 
+    expect(toggleButton).toHaveAttribute("aria-expanded", "true");
+
+    const popoverId = toggleButton.getAttribute("aria-controls");
+    const popover = canvas.getByRole("list");
+    expect(popover).toHaveAttribute("id", popoverId);
+
     const infoButton = canvas.getByLabelText("More Info");
     expect(infoButton).toHaveAttribute("aria-label", "More Info");
     expect(infoButton).toHaveAttribute("role", "button");
 
     await userEvent.tab();
-    expect(document.activeElement).toHaveAttribute("data-element", "main-button");
+    expect(document.activeElement).toHaveAttribute(
+      "data-element",
+      "main-button",
+    );
 
     await userEvent.tab();
-    expect(document.activeElement).toHaveAttribute("data-element", "toggle-button");
+    expect(document.activeElement).toHaveAttribute(
+      "data-element",
+      "toggle-button",
+    );
+
+    await userEvent.keyboard("{Escape}");
+    await userInteractionPause(300);
+    expect(toggleButton).toHaveAttribute("aria-expanded", "false");
```

Comment:
comment: I don't think we need to test this type of thing. We should be using play functions to capture the state of components after user interaction so that chromatic can snapshot the SplitButton expanded and we can capture the baseline for any future regressions